src/logic/operate.js

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable import/extensions */
+import Big from 'big.js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  const x = new Big(numberOne);
+  const y = new Big(numberTwo);
+
+  if (operation === '-') {
+    return x.minus(y);
+  }
+
+  if (operation === '+') {
+    return x.plus(y);
+  }
+
+  if (operation === 'X') {
+    return x.mul(y);
+  }
+
+  return x.div(y);
```

Comment:
dont forget to handle DIVISION BY ZERO on next milestone, because this will generate an error